src/plugins/elasticsearch.js

```diff
@@ -21,9 +21,15 @@ function createWrapRequest (tracer, config) {
         }
       })
 
-      if (JSON.stringify(params.body)) {
-        span.setTag('elasticsearch.body', JSON.stringify(params.body))
+      const indexPath = JSON.stringify(params.path).split('/')[1]
+
+      if (indexPath.substring(0, 1) !== '_') {
```

Comment:
ES automatically assigns urls to operations. The native operations are usually the first part of the URL and preceded with an underscore. eg _bulk, _cat. In that case, it will not be the index.

Reply:
Oh, didn't know that indices couldn't start w/ underscores.

Reply:
what is the significance of this check?